data/json/items/armor/brigandine

```diff
@@ -0,0 +1,180 @@
+    "warmth": 5,
+    "longest_side": "50 cm",
+    "material_thickness": 1.5,
+    "environmental_protection": 1,
+    "flags": [ "OUTER", "WATER_FRIENDLY" ],
+    "armor": [
+      {
+        "covers": [ "arm_l", "arm_r" ],
+        "encumbrance": 10,
+        "coverage": 95,
+        "specifically_covers": [ "arm_shoulder_l", "arm_shoulder_r" ]
+      }
+    ]
+  },
+  {
+    "id": "legguard_lc_brigandine",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "pair of mild splint leg guards", "str_pl": "pairs of mild splint leg guards" },
+    "description": "",
+    "weight": "3797 g",
+    "volume": "2500 ml",
+    "price": 40000,
+    "price_postapoc": 1000,
+    "to_hit": 1,
+    "material": [ "lc_steel", "wool" ],
+    "symbol": "[",
+    "looks_like": "legguard_lightplate",
+    "color": "light_gray",
+    "warmth": 10,
+    "material_thickness": 1.5,
+    "environmental_protection": 1,
+    "flags": [ "OUTER", "WATER_FRIENDLY", "ALLOWS_TAIL" ],
+    "armor": [
+      {
+      "material": [
+          { "type": "lc_steel", "covered_by_mat": 95, "thickness": 0.8 },
+          { "type": "wool", "covered_by_mat": 100, "thickness": 1.5 }
+        ],
+        "covers": [ "leg_l", "leg_r" ],
+        "encumbrance": 4,
+        "coverage": 95,
+        "cover_melee": 95,
+        "cover_ranged": 95,
+        "cover_vitals": 70,
+        "specifically_covers": [ "leg_lower_r", "leg_lower_l", "leg_upper_r", "leg_upper_l" ]
+      }
+    ]
+  },
+  {
+    "id": "lc_brigandine_hands",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "pair of mild steel brigandine gloves", "str_pl": "pairs of mild steel brigandine gloves" },
+    "description": "",
+    "weight": "575 g",
+    "volume": "750 ml",
+    "price": 5012,
+    "price_postapoc": 2500,
+    "to_hit": -1,
+    "bashing": 6,
+    "material": [ "leather", "lc_steel" ],
+    "symbol": "[",
+    "color": "light_red",
+    "flags": [ "STURDY", "OUTER" ],
+    "armor": [
+      {
+        "encumbrance": 7,
+        "coverage": 100,
+        "material": [
+          { "type": "lc_steel", "covered_by_mat": 100, "thickness": 1.0 },
+          { "type": "leather", "covered_by_mat": 100, "thickness": 1.5 }
+        ],
+        "covers": [ "hand_l", "hand_r" ],
+        "specifically_covers": [ "hand_back_l", "hand_back_r", "hand_wrist_l", "hand_wrist_r" ]
+      },
+      {
+        "encumbrance": 0,
+        "coverage": 100,
+        "material": [
+          { "type": "leather", "covered_by_mat": 100, "thickness": 1.5 }
+        ],
+        "covers": [ "hand_l", "hand_r" ],
+        "specifically_covers": [ "hand_fingers_l", "hand_fingers_r", "hand_palm_l", "hand_palm_r" ]
+      },
+      {
+        "encumbrance": 0,
+        "coverage": 50,
+        "cover_melee": 95,
+        "cover_ranged": 90,
+         "//": "only outer side if hand and fingers is covered by plates, but it shoul be very hard to hit unprotected part of you holdibg somthing in youre hands",
```

Comment:
```suggestion
         "//": "only outer side if hand and fingers is covered by plates, but it should be very hard to hit unprotected part if you're holding something in your hands",
```

Reply:
yeah, my english is not perfect, tanks for correcting this